content/_help/get-started/authentication-methods._en.md

```diff
@@ -57,7 +57,7 @@ We strongly recommend you add a second authentication method in case you change
 
 ## Authentication application
 
-Authentication applications are downloaded to your device and generate secure, six-digit codes you use to sign in to your accounts. While authentication applications are not protected if your device is lost or stolen, this method offers more security than phone calls or text messaging against phishing, hacking, or interception.
+Authentication applications are downloaded to your device and generate secure, six digit one-time codes you use to sign in to your accounts. While authentication applications are not protected if your device is lost or stolen, this method offers more security than phone calls or text messaging against phishing, hacking, or interception.
```

Comment:
This seems wrong to me. See discussion - https://gsa-tts.slack.com/archives/C01710KMYUB/p1731695242328959?thread_ts=1731689396.777079&cid=C01710KMYUB.